spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/sql/SparkStreamingQueryEventProcessor.scala

```diff
@@ -47,8 +47,8 @@ extends AbstractEventProcessor[QueryProgressEvent] with AtlasEntityUtils with Lo
       val path = e.progress.sink.description.substring(begin + 1, end)
       logDebug(s"record the streaming query sink output path information $path")
       outputEntity = external.pathToEntity(path)
-    } else if (e.progress.sink.description.contains("ConsoleSinkProvider")) {
-      logInfo(s"do not track the console output as Atlas entity ${e.progress.sink.description}")
+    } else {
+      logDebug(s"Atlas output entity was not able to made for ${e.progress.sink.description}")
       return
```

Comment:
If it doesn't return, output entity becomes `null` which throws an exception (which becomes a warning):

```
org.apache.atlas.AtlasServiceException:
Metadata service API org.apache.atlas.AtlasClientV2$API_V2@4f25616d failed with status 400 (Bad Request)
 Response Body ({"errorCode":"ATLAS-400-00-01A","errorMessage":"invalid parameters: found null entity"})
```